draft-ietf-quic-transport.md

```diff
@@ -3204,9 +3204,16 @@ response to further packets that it receives.
 A receiver MUST discard a newly unprotected packet unless it is certain that it
 has not processed another packet with the same packet number from the same
 packet number space. Duplicate suppression MUST happen after removing packet
-protection for the reasons described in Section 9.3 of {{QUIC-TLS}}. An
-efficient algorithm for duplicate suppression can be found in Section 3.4.3 of
-{{?RFC4303}}.
+protection for the reasons described in Section 9.3 of {{QUIC-TLS}}.
+
+Endpoints that track individual packets for the purposes of detecting duplicates
+might accumulate excessive state.  The data required for detecting duplicates
+can be limited by maintaining a minimum packet number below which all packets
+are immediately dropped.  Endpoints that use a minimum packet number need to
+ensure that any value does not cause delayed packets to be dropped.  Any minimum
```

Comment:
This doesn't apply to arbitrarily delayed packets, otherwise every minimum value will be too high.

Reply:
I think I'd drop this sentence.

Reply:
This was the entire point of the issue though.